app/javascript/react/components/Graph/Graph.tsx

```diff
@@ -56,9 +56,12 @@ const Graph: React.FC<GraphProps> = ({ streamId, sessionType }) => {
       : selectMobileStreamShortInfo
   );
 
+  const { minMeasurementValue, maxMeasurementValue } =
+    useSelector(selectFixedExtremes);
+
   const measurements = graphData?.measurements || [];
   const unitSymbol = streamShortInfo?.unitSymbol || "";
-  const measurementType = "Particulate Matter"; // take this parameter from filters in the future
+  const measurementType = "Particulate Matter";
```

Comment:
Maybe let's create some special constant for such a type? I think this name is important, is it?

Reply:
This is how it will be until we install filters.